rust/docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: "3"
+version: "3.4"
```

Comment:
This change snuck back in

Reply:
lol sorry I did a rebase and the change came back

Reply:
Yeah I think it's gonna need a new commit or something because the change is there